datalad/tests/utils.py

```diff
@@ -279,7 +279,10 @@ def _prep_file_under_git(path, filename):
             raise
 
     # path to the file within the repository
-    file_repo_dir = os.path.relpath(path, repo.path)
+    # repo.path is a "realpath" so to get relpath working correctly
+    # we need to relpath our path as well
```

Comment:
"relpath" <-> "realpath" in 283.
Otherwise confusing in this particular paragraf. ;-)

Reply:
```suggestion
    # we need to realpath our path as well
```